entrypoint/update-ampinstmgr.sh

```diff
@@ -22,10 +22,9 @@ else
 fi
 
 echo "Downloading latest ampinstmgr... (This can take ~5 minutes; CubeCoders limits their download speeds to a crawl.)"
-DL_PATH=/tmp/ampinstmgr.zip
-wget -nv http://cubecoders.com/Downloads/ampinstmgr.zip -O ${DL_PATH}
+wget -nv -P ${AMP_BIN_DIR} http://cubecoders.com/Downloads/ampinstmgr.zip 
```

Comment:
Why do you remove the `DL_PATH`? The reason I download to a `tmp` path is in case the container is shut down in the middle of download. Writing it to the volume will cause an unfinished download to linger. Perhaps we should add the `-c` (continue) argument to `wget` which should continue the interrupted download. What do you think?

Reply:
Yeah, your right with this one, but what i initially wanted to achieve is not downloading amp every time the container starts. Usually ampinstmgr shares the same version-numbering scheme with ampcore, so downloading both when one gets an update moght be better then downloading one when it updates an the second when the container restarts. So to make it persistent, i shifted it over to the volume. That might also help with testing, as you dont have to download it every time first.

I have to look a bit more into this first, but this was my initial intention.

Reply:
Ah I see, there's definitely room for optimization. How about we just add the `-c` flag, which will allow the download to continue if it fails, then we can leave it downloading where it is. Like I mentioned [here](https://github.com/MitchTalmadge/AMP-dockerized/pull/36#issuecomment-732456427) I would love to look into a different way of downloading ampinstmgr in the future, so we can come back to this!